SignalR_Server/Controllers/Game_Controller.cs

```diff
@@ -172,17 +179,30 @@ public void BeginNewQuestion(string gameKey)
 
         #region Get game value methods
 
-        // Returns the answer stats for the current hand
-        public M_AnswerStats GetHandAnswerStats(string gameKey)
-        {            
-            return GetGame(gameKey).GetAnswerStats();
+        // Returns the stats for the finished question
+        public M_AnswerStats GetQuestionStats(string gameKey)
+        {
+            return GetAnswerStats(GetGame(gameKey));
         }
         
         // Returns the game stats
         public IList<M_AnswerStats> GetGameStats(string gameKey)
         {
+            return GetGame(gameKey).GameAnswerStats;
+        }
+
+        public KeyValuePair<string, M_QuestionCard> GetFocusedPlayerIdAndQuestion(string gameKey)
```

Comment:
Jack, not exactly sure why you're using key-value pair for two unrelated objects. I mean, yes they're both focused, but I don't think the two should be linked to each other like this, unless there's some server magic that needs to happen that I don't understand.

Reply:
A good fix would be to separate the two into GetFocusedPlayerId and GetFocusedQuestion methods.

Reply:
Just noticed my suggestion is what you were doing previously. So I'm assuming you did this as a fix or an optimization. Either way, I'm still very skeptical of using a key-value pair for anything other than an explicit dictionary-esque relationship where the value *depends on* the key.

Just one obsessive compulsive's opinion.

Reply:
I actually just used that as a quick way to pass two different variables actually. It was more of a temporary fix.